mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb

```diff
@@ -892,6 +892,33 @@ func TestAccSqlDatabaseInstance_withPrivateNetwork_withAllocatedIpRangeClone(t *
 	})
 }
 
+func TestAccSqlDatabaseInstance_withPrivateNetwork_withEnablePrivatePathForGoogleCloudServices(t *testing.T) {
```

Comment:
On the topic of slow SQL tests, does this seem like a field that we could instead add to an existing test, so that we can avoid creating all of this infrastructure? Perhaps something like `testAccSqlDatabaseInstance_withPrivateNetwork_withoutAllocatedIpRange`?

Reply:
Yes, I have merged this test case into `testAccSqlDatabaseInstance_withPrivateNetwork_withoutAllocatedIpRange`: https://github.com/GoogleCloudPlatform/magic-modules/pull/6986/commits/9499f5b30dc511f1abdc74268d19556932d7abc8